src/DetailsView/reports/components/outcome-chip.tsx

```diff
@@ -13,7 +16,7 @@ interface OutcomeChipProps {
 
 export const OutcomeChip = NamedSFC<OutcomeChipProps>('OutcomeChip', props => {
     const { outcomeType, count } = props;
-    const { pastTense } = outcomeTypeSemantics[outcomeType];
+    const { pastTense } = outcomeTypeSemantics[outcomeType] || outcomeText[outcomeType];
```

Comment:
This doesn't work because `outcomeTypeSemantics` contains `{ pastTense: <string> }` but `outcomeText` just contains <string>, so we get `undefined` for `inapplicable`. (I also note that `outcomeTypeSemantics` returns a lowercase string but `outcomeText` returns a Capitalized string.)

Consider having outcomeTypeSementics support all `OutcomeType` values (not just `RequirementOutcomeType`) to simplify?